charts/pega/templates/_helpers.tpl

```diff
@@ -187,3 +187,14 @@ readinessProbe:
     Background
   {{- end -}}
 {{- end }}
+
+# Load balancer session cookie stickiness time in seconds,
+# calculated as sum of .requestor.passivationTimeSec and passivation delay.
+{{- define "lbSessionCookieStickiness" }}
```

Comment:
Is it possible to add a check after the to see if `alb_stickiness_lb_cookie_duration_seconds` is set and use that value if the new configuration is not present?  That will preserve backwards compatibility and not require client changes of their values.yaml files.

Reply:
Good idea. Added a check for alb_stickiness_lb_cookie_duration_seconds after checking that new configuration values are missing. If both new and old configs are present new will take precedence. Added one more check for presence of `requestor` block as it is missing in old configs and anyway it is optional.